__tests__/app.test.js

```diff
@@ -199,7 +199,19 @@ describe("GET /api/articles", () => {
         expect(articles[6]).toHaveProperty("comment_count", "11");
       });
   });
+  test("should allow filtering with topic query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        articles.forEach((article)=>{
+          expect(article.topic).toBe('mitch')
+        })
+      });
 });
+})
```

Comment:
2 tests missing here:

-  a valid topic query that has no articles (topic=paper) should respond with 200 and an empty array
-  a non-existent topic should respond with 404